charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -20,6 +20,7 @@ data:
         cluster: {{ .Release.Namespace }}
         project: {{ .Values.shoot.project }}
         shoot_name: {{ .Values.shoot.name }}
+        name: {{ .Values.shoot.name }}
```

Comment:
why not reusing `shoot_name`?

Reply:
I want to get rid of `shoot_name`. The `gardener-metrics-exporter` uses the label `name` for the shoot name. Once all data is aggregated, this causes problems when sometimes the label is `name` and other times `shoot_name` based on where the data came from.

Reply:
OK, thank you!